### ISSUE TITLE
[arm32/Linux] System.Resources.ResourceManager.Tests failures on arm32 linux

### ISSUE THREAD

**Issue by joperezr** [dotnet/corefx#33877]
Running the arm32 tests on linux for System.Resources.ResourceManager.Tests produced the following results: passed 94/96 tests.

Both test failures failed with the following callstack:
```
System.TypeInitializationException : The type initializer for 'Gdip' threw an exception.\n---- System.DllNotFoundException : Unable to load DLL 'libgdiplus': The specified module could not be found.
   at System.Drawing.SafeNativeMethods.Gdip.GdipCreateBitmapFromFile(String filename, IntPtr& bitmap)
   at System.Drawing.Bitmap..ctor(String filename, Boolean useIcm)
   at System.Drawing.Bitmap..ctor(String filename)
   at System.Resources.Tests.ResourceManagerTests.EnglishImageResourceData()+MoveNext() in /opt/corefx/src/System.Resources.ResourceManager/tests/ResourceManagerTests.cs:line 266
   at System.Linq.Enumerable.SelectEnumerableIterator`2.MoveNext()
----- Inner Stack Trace -----
   at System.Runtime.InteropServices.FunctionWrapper`1.get_Delegate()
   at System.Drawing.SafeNativeMethods.Gdip.GdiplusStartup(IntPtr& token, StartupInput& input, StartupOutput& output)
   at System.Drawing.SafeNativeMethods.Gdip..cctor()
```

**Comment by danmosemsft** [comment]
We should try to install libgdiplus.

**Comment by joperezr** [comment]
[testResults.zip](https://github.com/dotnet/corefx/files/2654994/testResults.zip)

**Comment by joperezr** [comment]
Yup, this one looks like it's just a problem of my pi missing a dependency.

**Comment by danmosemsft** [comment]
That was only happened to be exposed because these tests were not protected by a check for libgdiplus.

Same for some other platforms:
https://mc.dot.net/#/product/netcore/30/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Fcli~2F/build/20181206.01/workItem/System.Resources.ResourceManager.Tests/analysis/xunit/System.Resources.Tests.ResourceManagerTests~2FGetResourceSet_Images

You can just put `[ConditionalTheory(Helpers.IsDrawingSupported)]` on these tests.

It would be nice to put libgdiplus on the device, if it's avialable for ARM, so it gets coverage.

**Comment by wfurt** [comment]
This should be fixed by #34195. Drawing tests use IsDrawingSupported but resource manager did not. That was also causing test failures on Alpine and FreeBSD.
